osu.Game.Rulesets.Osu.Tests/TestSceneSliderSnaking.cs

```diff
@@ -72,7 +65,7 @@ public void TestSnakingEnabled(int sliderIndex)
         {
             AddStep("enable autoplay", () => autoplay = true);
             base.SetUpSteps();
-            AddUntilStep("wait for track to start running", () => track.IsRunning);
+            AddUntilStep("wait for track to start running", () => MusicController.IsPlaying);
```

Comment:
Is there a reason this is still using `MusicController`? There's tests that use both `Beatmap.Value.Track` and `MusicController` and it feels kinda weird... I'm going to mark down all tests that have `MusicController`, since I believe the idea was to move back to `Beatmap.Value.Track`.

Reply:
This is another weird one. Can't revert because the loading is being done by the music controller in the base scene. Will have to check where the track is valid to be fetched.

Reply:
Actually seems okay.